src/main/java/com/example/learninghub/course/CourseController.java

```diff
@@ -22,4 +24,20 @@ public CourseController(CourseService courseService) {
     public String get() {
```

Comment:
This can be removed.

Reply:
Deleted in [d1998f1](https://github.com/Lukasz-gro/learning-hub-server/pull/10/commits/d1998f168ed5e3cb7ea96de92db7e855ea9ba3b7)